aries-site/src/examples/components/tag/CreateTagSimple.js

```diff
@@ -0,0 +1,152 @@
+      <Form
+        messages={{ required: 'This is a required field.' }}
+        onSubmit={() => {
+          const exists = currentTags.filter(t => t === value);
+          if (!exists.length) setCurrentTags([...currentTags, value]);
+          setValue(defaultValue);
+        }}
+      >
+        <Grid
+          columns={
+            size !== 'small'
+              ? [['auto', 'medium'], ['auto', 'medium'], 'auto']
+              : '100%'
+          }
+          gap={{ column: 'medium', row: 'small' }}
+        >
+          <FormField
+            name="tag-value-simple"
+            htmlFor="tag-value-simple"
+            label="Value"
+            required={{ indicator: false }}
+          >
+            <Select
+              name="tag-value-simple"
+              id="tag-value-simple"
+              placeholder="Select or create a value"
+              value={value}
+              options={valueOptions}
+              onChange={({ option }) => {
+                if (option.includes(prefix)) {
+                  defaultValueOptions.pop(); // remove Create option
+                  defaultValueOptions.push(valueSearch);
+                  setValueOptions(alphabetize(defaultValueOptions));
+                  setValue(valueSearch);
+                } else {
+                  setValue(option);
+                }
+              }}
+              onSearch={text => {
+                const nextValueOptions = defaultValueOptions;
+                updateCreateOption(text, nextValueOptions);
+                const exp = getRegExp(text);
+                setValueOptions(
+                  alphabetize(nextValueOptions.filter(o => exp.test(o))),
+                );
+                setValueSearch(text);
+              }}
+            />
+          </FormField>
+          <Box
+            justify="start"
+            // use theme style values to align Button with FormField input
+            pad={
+              size !== 'small'
+                ? {
+                    top: `${parseInt(theme.text.xsmall.height, 10) +
+                      parseInt(
+                        theme.global.edgeSize[theme.formField.label.margin.top],
+                        10,
+                      ) +
+                      parseInt(
+                        theme.global.edgeSize[
+                          theme.formField.content.margin.vertical
+                        ],
+                        10,
+                      )}px`,
+                  }
+                : undefined
+            }
+          >
+            <Button
+              alignSelf="start"
+              label="Assign"
+              type="submit"
+              fill={size === 'small' ? 'horizontal' : undefined}
+              secondary
+            />
+          </Box>
+        </Grid>
+      </Form>
+      <TagResults>
+        {currentTags.length === 0 ? (
+          <Text margin="xsmall">No tags have been assigned.</Text>
+        ) : (
+          currentTags.map((t, index) => (
```

Comment:
The button alignment seems to off. Riding too high.

![Screen Shot 2022-01-10 at 2 15 24 PM](https://user-images.githubusercontent.com/1756948/148840822-2df3d030-ff13-41ae-aa7e-469c9f1ca09f.png)

Reply:
Hmmm -- I wonder if this is a bug introduced by https://github.com/grommet/grommet/pull/5856

Reply:
Could be. Curious that it doesn't show up on https://storybook.grommet.io/?path=/story/type-tag-onremove--on-remove... Do we have something intermediary in the theme?

Reply:
Adjusted vertical-align CSS for plainStyle: https://github.com/grommet/grommet/blob/4eb84fa581b14bbae2e75c3e5adfb4f7dd871354/src/js/components/Button/StyledButtonKind.js#L207

Didn't show up in the story because Grommet is using Button whereas HPE Theme uses ButtonKind.